macros/create_source_tables.sql

```diff
@@ -0,0 +1,41 @@
+{% macro create_source_tables(vocab_tables) %}
```

Comment:
this combines create_synthea_tables and create_vocab_tables into a single macro.